src/auth/basic/UserRequest.cc

```diff
@@ -168,7 +168,8 @@ Auth::Basic::UserRequest::HandleReply(void *data, const Helper::Reply &reply)
 
     // add new helper kv-pair notes to the credentials object
     // so that any transaction using those credentials can access them
-    r->auth_user_request->user()->notes.appendNewOnly(&reply.notes);
+    static const std::vector<std::string> appendKeys = { "group", "tag" };
```

Comment:
Make this a vector of SBuf and initialize with { SBuf("group"), SBuf("tag") }

Reply:
Done!